charts/linkerd2-multicluster-remote-setup/templates/service-mirror-rbac.yaml

```diff
@@ -12,10 +12,13 @@ metadata:
   annotations:
     {{.Values.createdByAnnotation}}: {{default (printf "linkerd/helm %s" .Values.linkerdVersion) .Values.cliVersion}}
 rules:
-- apiGroups:
-  - ""
-  resources: ["services", "configmaps"]
+- apiGroups: [""]
+  resources: ["services"]
   verbs: ["list", "get", "watch"]
+- apiGroups: [""]
+  resources: ["configmaps"]
+  verbs: ["get"]
+  resourceNames: ["linkerd-config"]  
```

Comment:
I think this grants permission to read a configmap in any namespace as long as it is named "linkerd-config".  This is probably fine and I don't know that we really care that much, but just for the sake of discussion: I think the most "correct" way to do this would be to also create a Role and RoleBinding in the linkerd namespace that allow access to just the linkerd-config configmap in that namespace specifically.